src/styles/Base/theme.scss

```diff
@@ -0,0 +1,38 @@
+/* Resets */
+
+ul {
+
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+
+  text-decoration: none;
+  color: #fff;
+}
+
+
+/* Colors */
+
+$main-black: #1F1C17;
```

Comment:
No need for the `main-` prefix. Just use `$black` cause thats OUR "black" color ;)
Do the same for the other variables.